### PR TITLE
[IMP] account: Legal note translatable

### PR DESCRIPTION
**Pull request by malb-odoo**
This commit will allow the legal notes on taxes to be translatable.

task: 4402997

---
I confirm I have signed the CLA and read the PR guidelines at www.odoo.com/submit-pr

### CODE REVIEW

File: addons/account/models/account_tax.py [hunk 1]
```diff
@@ -194,7 +194,7 @@ class AccountTax(models.Model):
     country_code = fields.Char(related='country_id.code', readonly=True)
     is_used = fields.Boolean(string="Tax used", compute='_compute_is_used')
     repartition_lines_str = fields.Char(string="Repartition Lines", tracking=True, compute='_compute_repartition_lines_str')
-    invoice_legal_notes = fields.Html(string="Legal Notes", help="Legal mentions that have to be printed on the invoices.")
+    invoice_legal_notes = fields.Html(string="Legal Notes", translate=True, help="Legal mentions that have to be printed on the invoices.")
```
Comment (epictete) [inline addons/account/models/account_tax.py:197]:
```suggestion
    invoice_legal_notes = fields.Html(string="Legal Notes", translate=html_translate, help="Legal mentions that have to be printed on the invoices.")
```

Comment (malb-odoo) [reply addons/account/models/account_tax.py:197]: Is that new ? :exploding_head:

Comment (epictete) [reply addons/account/models/account_tax.py:197]: Not really :nerd_face: 
https://github.com/odoo/odoo/commit/6c8e53a08705b948833fa58d3c5dc182ebb889c4

Comment (malb-odoo) [reply addons/account/models/account_tax.py:197]: didn't know that, i will change asap

**Review by epictete — APPROVED** [review]
@robodoo r+

**Comment by Xavier-Do** [comment]
@robodoo r-

Looks related to the failure in https://runbot.odoo.com/runbot/build/72325723

File: addons/account/models/account_tax.py (hunk 1 above)
Comment (william-andre) [reply addons/account/models/account_tax.py]: But it's the default if you set `translate=True`, from the same commit...

**Review by william-andre — APPROVED** [review]
@robodoo r+

**Comment by william-andre** [comment]
@robodoo retry

**Pull request closed without merging**